samples/modules/lvgl/demos/sample.yaml

```diff
@@ -89,6 +93,8 @@ tests:
     extra_args:
       - SHIELD=rtklcdpar1s00001be
     extra_configs:
-      - CONFIG_LV_DEMO_MUSIC_LANDSCAPE=y
+      - CONFIG_LV_Z_DEMO_WIDGETS=y
```

Comment:
Not bent on any particular demo, but any reason to use the widgets one specifically?